scripts/_common.sh

```diff
@@ -83,19 +83,19 @@ install_jellyfin_packages() {
 	ynh_setup_source --dest_dir=$tempdir --source_id="web.$debian.$YNH_ARCH"
 
 	# Install the packages
+	ynh_exec_warn_less dpkg --force-confdef --force-confnew -i $tempdir/jellyfin-web.deb
 	ynh_exec_warn_less dpkg --force-confdef --force-confnew -i $tempdir/jellyfin-ffmpeg5.deb
 	ynh_exec_warn_less dpkg --force-confdef --force-confnew -i $tempdir/jellyfin-server.deb
-	ynh_exec_warn_less dpkg --force-confdef --force-confnew -i $tempdir/jellyfin-web.deb
 
```

Comment:
Ooooh it was that simple! :D

Reply:
it's what I think...
when reading the log when install fail:
`[ERR] The server is expected to host the web client, but the provided content directory is either invalid or empty: /usr/share/jellyfin/web. If you do not want to host the web client with the server, you may set the '--nowebclient' command line flag, or set'hostwebclient=false' in your config settings.`